src/app/collection/shared/included/collection-learning-object-card/collection-learning-object-card.component.scss

```diff
@@ -0,0 +1,89 @@
+@import '../../../collection-globals.scss';
+.container{
+    width: 20vw;
+    margin-bottom: 50px;
+    display: flex;
+}
+
+
+
+.header{
+    &.module{
+        background-image: url("../../../../../assets/images/pexels-tima-miroshnichenko-5380642.jpg");
+        margin: 0 15px;
+    }
+    &.micromodule{
+        background-image: url('../../../../../assets/images/adi-goldstein-EUsVwEOsblE-unsplash\ 2.png');
+        margin: 0 15px;
+    }
+    &.nanomodule{
+        background-image: url("../../../../../assets/images/pexels-tima-miroshnichenko-5380618.jpg");
+        margin: 0 15px;
+    }
+    &.unit{
+        background-image: url("../../../../../assets/images/michael-dziedzic-nLFqr9Mr9H8-unsplash\ 1.png");
+        margin: 0 15px;
```

Comment:
The margin can get moved out to the header class and you won't have to repeat it in each of the nested length classes.